src/CommonDBTM.php

```diff
@@ -5677,6 +5677,17 @@ private function assetBusinessRules($condition)
                 $input['_auto'] = 0;
             }
 
+            //if agent exist pass the 'tag' to RuleAssetCollection
+            if (
+                Toolbox::hasTrait($this, \Glpi\Features\Inventoriable::class)
+                && method_exists($this, 'getInventoryAgent')
+            ) {
+                $agent = $this->getInventoryAgent();
+                if ($agent !== null) {
+                    $input['tag'] = $agent->fields['tag'];
```

Comment:
We used to add a `_`  prefix to "virtual" fields (i.e. fields that are not in DB).

```suggestion
                    $input['_tag'] = $agent->fields['tag'];
```